modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -773,17 +854,16 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
     assert.equal(serializedHtml, html, 'Should be html with base64 uri retained');
   });
 
-  // TODO: TINY-4627/TINY-8202: figure out why dompurify returns a different dom structure to us
-  it.skip('TINY-7756: Parsing invalid nested children', () => {
-    const parser = DomParser();
-    const html = '<table><button><a><meta></meta></a></button></table>';
+  it('TINY-7756: Parsing invalid nested children', () => {
+    const schema = Schema({ valid_children: '-td[button|a|div]' });
+    const parser = DomParser({}, schema);
+    const html = '<table><tr><td><button><a><meta /></a></button></td></tr></table>';
```

Comment:
Had to change this a little as the native parser will also move the element outside if it's not valid. As such, this now uses valid HTML but changes the schema to get the invalid nested children that we needed to test.